test/edwadrds_point_test.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:convert/convert.dart';
+import 'package:solana/src/edwards25519/compressed_edwards_y.dart';
+import 'package:solana/src/edwards25519/edwards_point.dart';
+import 'package:solana/src/edwards25519/field_element.dart';
+import 'package:test/test.dart';
+
+EdwardsPoint ed25519BasePoint = EdwardsPoint(
+  FieldElement([
+    -14297830,
+    -7645148,
+    16144683,
+    -16471763,
+    27570974,
+    -2696100,
+    -26142465,
+    8378389,
+    20764389,
+    8758491
+  ]),
```

Comment:
There's a little trick in the formatting of long lists. You can add comment to the first line, and it will respect line breaks in other lines, so it will be more compact:

```suggestion
  FieldElement([
    -14297830, -7645148, 16144683, -16471763, 27570974, -2696100, //
    -26142465, 8378389, 20764389, 8758491
  ]),
```

Reply:
Thank you! I didn't find anything about this ...